### PR TITLE
fix: value > vote, loosen delete protection, omit votes from search

### PR DESCRIPTION
**Pull request by jgravois**
* ensure that upvotes and downvotes aren't returned by search
* replace fieldname `value` with `vote`
* ensure that users can delete their own vote
* more descriptive `<meta>` tag in docs

@lisastaehli let me know if the second item in the list above is problematic. if you don't want to blow away your existing service entirely you should be able to add `vote` as an additional field in the ArcGIS Online UI. i'm happy to help if you run into any trouble.

AFFECTS PACKAGES:
@esri/hub-annotations

RESOLVES:
#123, #124, #125, #118

### CODE REVIEW

File: docs/src/_layout.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, user-scalable=no">
     <title>{{title if title}}{{" | " + titleSegments | join(" |") if titleSegments}}{{" | " if title}}@esri/hub.js</title>
     {% if description %}
-    <meta name="description" content="This is an example of a meta description. This will often show up in search results.">
+    <meta name="description" content="A modular, high quality toolkit for working with <a href='https://hub.arcgis.com/'>ArcGIS Hub</a>">
```
Comment (tomwayson) [inline docs/src/_layout.html]: can meta tags contain anchor tags?

Comment (jgravois) [reply docs/src/_layout.html]: > The value must be a free-form string that describes the page ... There must not be more than one `meta` element where the `name` attribute value is an ASCII case-insensitive match for `description` per document.

https://html.spec.whatwg.org/multipage/semantics.html#meta-description

i'm pretty sure that means "no" 🤔

**Comment by lisastaehli** [comment]
I tried this and voting works fine, however the `outStatistics` are still on the wrong field `value`, so you would to change that as well to `vote` in order to fetch and summarize the votes correctly.

image

**Comment by jgravois** [comment]
> `outStatistics` are still on the wrong field `value`

sorry about that @lisastaehli! i've released [`v1.7.2`](https://github.com/Esri/hub.js/releases/tag/v1.7.2) with a fix.

**Comment by lisastaehli** [comment]
Awesome :) thanks. Now everything is working fine and we will migrate to 1.7.2

**Pull request merged**